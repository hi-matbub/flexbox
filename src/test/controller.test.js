import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ControlPanel from '../comps/controller/controller_.jsx';

it('renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<ControlPanel />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders snapshot as expected', () => {
  const tree = renderer.create(<ControlPanel />).toJSON();
  expect(tree).toMatchSnapshot();
});
