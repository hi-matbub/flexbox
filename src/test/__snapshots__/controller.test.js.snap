// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot as expected 1`] = `
<div
  className="control"
>
  <button
    act_status={false}
    className="control_switch"
    onClick={[Function]}
    style={
      Object {
        "background": "#c8c8c8",
        "border": "none",
        "borderRadius": "6px",
        "cursor": "pointer",
        "margin": "2px",
      }
    }
    type="button"
  />
  <button
    act_status={false}
    className="control_switch"
    onClick={[Function]}
    style={
      Object {
        "background": "#c8c8c8",
        "border": "none",
        "borderRadius": "6px",
        "cursor": "pointer",
        "margin": "2px",
      }
    }
    type="button"
  />
  <button
    act_status={false}
    className="control_switch"
    onClick={[Function]}
    style={
      Object {
        "background": "#c8c8c8",
        "border": "none",
        "borderRadius": "6px",
        "cursor": "pointer",
        "margin": "2px",
      }
    }
    type="button"
  />
  <button
    act_status={false}
    className="control_switch"
    onClick={[Function]}
    style={
      Object {
        "background": "#c8c8c8",
        "border": "none",
        "borderRadius": "6px",
        "cursor": "pointer",
        "margin": "2px",
      }
    }
    type="button"
  />
  <button
    act_status={false}
    className="control_switch"
    onClick={[Function]}
    style={
      Object {
        "background": "#c8c8c8",
        "border": "none",
        "borderRadius": "6px",
        "cursor": "pointer",
        "margin": "2px",
      }
    }
    type="button"
  />
  <button
    act_status={false}
    className="control_switch"
    onClick={[Function]}
    style={
      Object {
        "background": "#c8c8c8",
        "border": "none",
        "borderRadius": "6px",
        "cursor": "pointer",
        "margin": "2px",
      }
    }
    type="button"
  />
  <button
    act_status={false}
    className="control_switch"
    onClick={[Function]}
    style={
      Object {
        "background": "#c8c8c8",
        "border": "none",
        "borderRadius": "6px",
        "cursor": "pointer",
        "margin": "2px",
      }
    }
    type="button"
  />
  <button
    act_status={false}
    className="control_switch"
    onClick={[Function]}
    style={
      Object {
        "background": "#c8c8c8",
        "border": "none",
        "borderRadius": "6px",
        "cursor": "pointer",
        "margin": "2px",
      }
    }
    type="button"
  />
  <button
    act_status={false}
    className="control_switch"
    onClick={[Function]}
    style={
      Object {
        "background": "#c8c8c8",
        "border": "none",
        "borderRadius": "6px",
        "cursor": "pointer",
        "margin": "2px",
      }
    }
    type="button"
  />
</div>
`;
